src/apps/loan-list/modal/renew-loans-modal-content.tsx

```diff
@@ -9,21 +9,29 @@ import {
   getAmountOfRenewableLoans
 } from "../../../core/utils/helpers/general";
 import { FaustId } from "../../../core/utils/types/ids";
+import useIsInViewport from "../../../components/utils/in-viewport";
+import { useText } from "../../../core/utils/text";
 
 interface RenewLoansModalContentProps {
   renewable: number | null;
   loansModal: LoanV2[];
   buttonLabel: string;
   checkboxLabel: string;
+  buttonBottomLabel: string;
+  checkboxBottomLabel: string;
 }
 
 const RenewLoansModalContent: FC<RenewLoansModalContentProps> = ({
   renewable,
   loansModal,
   checkboxLabel,
-  buttonLabel
+  buttonLabel,
+  buttonBottomLabel,
+  checkboxBottomLabel
 }) => {
+  const t = useText();
```

Comment:
Please remember to remove `t` if it is not used.

Reply:
Definitely, will fix